notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb

```diff
@@ -155,7 +155,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from starfish.pipeline.filter import Filter\n",
+    "from starfish.image._filter import Filter\n",
```

Comment:
We don't want people to have to import hidden modules like this.

I see that starfish.image is importing Filter, so can we just do `from starfish.image import Filter`?

Reply:
We sure can.